plugin/adsk/plugin/plugin.cpp

```diff
@@ -108,8 +108,8 @@ MStatus initializePlugin(MObject obj)
     }
 #endif
 
-    MGlobal::executeCommand("source \"mayaUsdMenu.mel\"");
-    MGlobal::executeCommand("mayaUsdMenu_loadui");
+    plugin.registerUI("mayaUsd_pluginUICreation", "mayaUsd_pluginUIDeletion", 
+        "mayaUsd_pluginBatchLoad", "mayaUsd_pluginBatchUnoad");
 
```

Comment:
Moving anything we do on load and unload into the MEL proc's that are now registered with the plugin.  For now that's the menu creation + autoload of the ufeSupport plugin.

I am also leaving in empty callbacks for when you load/unload in batch mode.  Better to have the scripts in place if/when we need them instead of someone not realizing that MFnPlugin does have separate callbacks in those situations and then trying to code something specific in another place.